packages/admin/admin/src/form/fields/NumberField.tsx

```diff
@@ -0,0 +1,8 @@
+import * as React from "react";
+
+import { Field, FieldProps } from "../Field";
+import { FinalFormNumberInput } from "../FinalFormNumberInput";
+
+export const NumberField = ({ ...restProps }: FieldProps) => {
```

Comment:
This. should probably have it's own props type, like for example the `TextField`.